src/common/replayBrowser/dao.ts

```diff
@@ -0,0 +1,183 @@
+import { FileResult } from "common/replayBrowser";
+import { app } from "electron";
+import path from "path";
+import * as sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(path.join(app.getPath("userData"), "statsdb.sqlite"), (err) => {
+  if (err) {
+    return console.error(err.message);
+  }
+  console.log("Connected to the replay database.");
+  db.run(`
+  CREATE TABLE IF NOT EXISTS replays (
+       fullPath      TEXT PRIMARY KEY,
+       name          TEXT,
+       folder        TEXT,
+       startTime     TEXT,
+       lastFrame     INTEGER,
+       playerCount   INTEGER,
+       player1       INTEGER,
+       player2       INTEGER,
+       player3       INTEGER,
+       player4       INTEGER,
```

Comment:
What are are these values for? It would be good if you could elaborate a bit on your database design decisions as either a reply or in the PR description.

Reply:
I would personally prefer having a table which only keeps track of file information (e.g. `fullPath`, `name`, `folder`), and then separates table for storing actual game information, keyed by full path, rather than everything in a single table.

Reply:
See my post on Discord. However looking at it, they should be TEXT fields!!! It's funny because ts was managing to convert them properly so all queries where working as expected 🤦